temporalio/client.py

```diff
@@ -1506,6 +1663,295 @@ class WorkflowExecutionStatus(IntEnum):
     )
 
 
+class WorkflowExecutionAsyncIterator:
+    """Asynchronous iterator for :py:class:`WorkflowExecution` values.
+
+    Most users should use ``async for`` on this iterator and not call any of the
+    methods within. To consume the workflows into histories, call
```

Comment:
Nit: Into reads slightly confusing here
```suggestion
    methods within. To consume the workflows as histories, call
```

Reply:
Updated